src/Symfony/Component/Intl/Data/Generator/RegionDataGenerator.php

```diff
@@ -34,6 +34,8 @@ class RegionDataGenerator extends AbstractDataGenerator
     const BOUVET_ISLAND_ID = 'BV';
     const HEARD_MCDONALD_ISLANDS_ID = 'HM';
     const CLIPPERTON_ISLAND_ID = 'CP';
+    const EUROZONE_UNION_ID = 'EZ';
```

Comment:
actually, this should be named EUROZONE_ID I suppose

Reply:
fixed. I found in wiki that EZ is a **monetary** union, that's why I added union in constant name :slightly_smiling_face:  but yep, finally it's not a usual "union".